pipeline/outputmodules/graph.go

```diff
@@ -172,6 +172,13 @@ func computeStages(mods []*pbsubstreams.Module) (stages ExecutionStages) {
 				}
 			}
 
+			//Check block index dependence
```

Comment:
I'll have to double check that with you.. I'm not certain if all cases are covered or if some modules may be pushed down another stage because it wasn't ordered correctly..